go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/hashicorp/terraform-plugin-docs v0.14.1
 	github.com/hashicorp/terraform-plugin-framework v1.5.0
 	github.com/hashicorp/terraform-plugin-testing v1.6.0
-	github.com/vantage-sh/vantage-go v0.0.12
+	github.com/vantage-sh/vantage-go v0.0.14-0.20240406212905-64e885583c48
```

Comment:
eh?

Reply:
once upstream stuff is merged + released, will re-point

(see https://github.com/vantage-sh/vantage-go/pull/12)